core/src/main/java/de/muenchen/allg/itd51/wollmux/OpenExt.java

```diff
@@ -245,18 +242,21 @@ public void setSource(URL url)
   }
 
   /**
-   * Legt doc als die Datei fest mit der die externe Andwendung gestartet werden
-   * soll. Die Datei wird immer zuerst in eine temporäre Datei exportiert bevor die
-   * externe Anwendung aufgerufen wird. In welchem Format gespeichert wird, bestimmt
-   * die FILTER-Angabe in der Definition der externen Anwendung. Ob die Datei als
```

Comment:
Please leave uppercase words like FILTER, DOWNLOAD intact. This refers to the wollmux.conf syntax and must not be changed.